CONTENT/schedule.markdown

```diff
@@ -9,12 +9,12 @@ comment: goal - event emitter + bookmark bar app to subscribe
 
 ---
 [](#english)
-# Sessions of Wizardry
+# Magic sessions
```

Comment:
Ok, we can change back. Wizardry just sounds more complicated, but now we have german version so english speakers understand it without problems...